package.json

```diff
@@ -33,11 +34,14 @@
   },
   "devDependencies": {
     "@biomejs/biome": "1.8.3",
+    "@types/node": "^22.10.5",
     "@types/react": "^18.2.15",
     "@types/react-dom": "^18.2.7",
     "@vitejs/plugin-react": "^4.0.3",
     "glob": "^11.0.1",
     "typescript": "^5.0.2",
+    "jsdom": "^26.0.0",
```

Comment:
seems unused

Reply:
I just realized I "lost" the EthGlobalImporter somewhere along the way during rebasing and force pushes :scream: I restored the correct content where JSDOM is being used in EthGlobalImporter.